service/src/main/java/com/codeforcommunity/processor/BlocksProcessorImpl.java

```diff
@@ -47,8 +47,8 @@ public BlockResponse reserveBlocks(JWTData jwtData, List<String> blockIds) {
     updateDatabaseBlocks(eligibleBlocks, BlockStatus.RESERVED, jwtData.getUserId());
 
     List<String> successfulBlockIds =
-        brs.getOrDefault(BlockStatus.OPEN, db.newResult(BLOCK)).map(BlockRecord::getFid);
-    mapRequester.updateStreets(successfulBlockIds, BlockStatus.RESERVED);
+        brs.getOrDefault(BlockStatus.OPEN, db.newResult(BLOCK)).map(BlockRecord::getId);
```

Comment:
Tried to do this in the least destructive way possible. We basically pull FID before each update, so I just replaced that with ID. Open to other ways.